abs.yaml

```diff
@@ -0,0 +1,3 @@
+upload_without_merge: True
+channels:
+  mesters_org: conda_standalone
```

Comment:
FYI (in case you aren't aware), we now have staging channels and the builds automatically upload all artifacts to the staging channels. The artifact for your PR can be found at https://staging.continuum.io/prefect/fs/pyinstaller-feedstock. There is one channel per PR+commit. You can then reference the staging channels using `channels`.

Reply:
I wasn't aware. I tried looking at our internal documentation, but didn't find references to that in here: https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Building_out_dependency_chains.md

Reply:
Yeah, I'm not too sure where it's documented.